addons/test_mail_full/tests/test_mail_performance.py

```diff
@@ -253,21 +254,31 @@ def test_portal_message_format_norating(self):
                 format_res['attachment_ids'],
                 [
                     {
-                        'access_token': message.attachment_ids[0].access_token,
+                        'as_author_access_token': limited_field_access_token(
+                            message.attachment_ids[0], 'as_author_access_token'
+                        ),
```

Comment:
Is current user author of this message? This should likely not be there.

Make sure this test has at least one message for which current user is author, and one for which it isn't, so we can see the difference.

Reply:
Yes, it's here by mistake because I didn't restrict the presentation of the value in the portal formatter (my bad). Changing this test to what you said results in changing so many lines of this test class which is not related to this PR. Instead, I defined a very small test to check the result of the formatter in case of self author (although all of them will be deleted when replacing the fetch route).